django/core/cache/backends/filebased.py

```diff
@@ -13,13 +13,14 @@
 from django.core.files.move import file_move_safe
 
 
-def _write_content(f, expiry, value):
-    f.write(pickle.dumps(expiry, pickle.HIGHEST_PROTOCOL))
-    f.write(zlib.compress(pickle.dumps(value, pickle.HIGHEST_PROTOCOL)))
+def _write_content(f, expiry, value, protocol):
```

Comment:
Might make sense to rework this function as method.

Reply:
@sir-sigurd agreed!

Reply:
@sir-sigurd, @claudep handled in 1a1ed064635fd43fb9018e5d30af9452b337cdc5.